testapp/app.js

```diff
@@ -12,19 +11,8 @@ import routes from './routes.js'
 const app = express()
 let port = 3000
 
-// NHS.UK frontend package path
-const frontendPath = fileURLToPath(
-  dirname(import.meta.resolve('nhsuk-frontend/package.json'))
-)
-
 // Nunjucks configuration for application
-const appViews = [
-  join(import.meta.dirname, 'views'),
-  join(frontendPath, 'dist/nhsuk/components'),
-  join(frontendPath, 'dist/nhsuk/macros'),
-  join(frontendPath, 'dist/nhsuk'),
-  join(frontendPath, 'dist')
-]
+const appViews = [join(import.meta.dirname, 'views')]
```

Comment:
This last one could move too, but perhaps it should be configurable, in case you want to use a different folder in your prototype from `views/`?